test/linkedListLib.test.js

```diff
@@ -0,0 +1,267 @@
+const BigNumber = web3.BigNumber;
+const LinkedListMock = artifacts.require('LinkedListMock.sol');
+
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+const NULL = '0x0000000000000000000000000000000000000000';
+const HEAD = '0x0000000000000000000000000000000000000000';
+const PREV = false;
+const NEXT = true;
+
+const ADDRESSES = [
```

Comment:
These addresses can come from the second params of the `contract` block.

`contract('MyContract', [account1, account2, account3, account4, etc], => {...`